ingreedypy.py

```diff
@@ -203,7 +206,10 @@ class Ingreedy(NodeVisitor):
         / "fl"
 
         gallon
-        = "gallons"
+        = "gallons."
+        / "gallons"
+        / "gallon(s)"
+        / "gallon."
```

Comment:
Although I understand that some imperial units were in the past often followed by a full-stop, I'm not sure the value of catching a few historically-formatted documents is worthwhile for this library.  Most modern writing style guides - both scientific and journalistic - that I've found recommend _not_ placing a full-stop after a unit name.